codalab/lib/bundle_store.py

```diff
@@ -74,11 +54,320 @@ def get_bundle_location(self, data_hash):
         """
         pass
 
-    def reset(self):
+class BalancedMultiDiskBundleStore(BaseBundleStore, BundleStoreCleanupMixin, BundleStoreHealthCheckMixin):
```

Comment:
Why don't we just replace the hash ring implementation with `BalancedMultiDiskBundleStore`?